src/virt-install

```diff
@@ -241,7 +241,7 @@ try:
                           location_arg,
                           "--disk=path={},cache=unsafe".format(args.dest),
                           "--initrd-inject={}".format(ks_tmp.name),
-                          "--extra-args", "ks=file://{} console=tty0 console=ttyS0,115200n8 inst.cmdline inst.notmux".format(os.path.basename(ks_tmp.name))])
+                          "--extra-args", "ks=file://{} console=tty0 console=ttyS0,115200n8 console=ttyAMA0 inst.cmdline inst.notmux".format(os.path.basename(ks_tmp.name))])
```

Comment:
This one needs arch detection.

Reply:
In the end it was not needed, `ttyS0` works as well.